egs/fame/s5/local/score.sh

```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
```

Comment:
do not copy that file -- simple symlinking from steps/score_kaldi.sh was the intended use.